pkg/ccl/backupccl/backup_planning.go

```diff
@@ -411,19 +413,27 @@ func backupPlanHook(
 			}
 			prevBackups = make([]BackupManifest, len(incrementalFrom))
 			for i, uri := range incrementalFrom {
```

Comment:
_tiny_ nit: I usually just use the index version of `range` in these types of cases, so I only have to copy that one var for the closing over thing (and then inside the fn I just do `slice[j]`). You can also shadow `i := i` to be sure no one accidentally closes over the loop var again.

Reply:
Nice - thanks for the tip! I especially like shadowing `i` in this case.